src/components/widgets-settings/attach-kpi/attach-kpi.tmpl.html

```diff
@@ -0,0 +1,10 @@
+<div class="attach-panel" ng-if="$ctrl.showPanel">
+  <div class="attach-panel-text">
+    <span class="panel-text-label">Set target threshold: </span>
```

Comment:
Would be cool if it could be passed as an argument to the component.
In our current case I would put something like: "Get alerted when the cash projection goes below:"